### ISSUE TITLE
Unable to resolve module `AccessibilityInfo`

### ISSUE THREAD

**Issue by shihabus** [facebook/react-native#20824]
From the latest version of RN (0.55.6), this error is happening. Downgrading to RN 0.55.4 is a workaround but need to solve this issue asap, because downgrading isn't an optimal option.
